libcalico-go/lib/backend/watchersyncer/watchersyncer.go

```diff
@@ -167,8 +169,6 @@ func (ws *watcherSyncer) run(ctx context.Context) {
 		// call again.
 		updates = ws.sendUpdates(updates)
 	}
-
-	ws.wgws.Done()
```

Comment:
Don't we still need this Done()? 

We Add() 1, plus one for each watcher syncer, but we only call Done() for each WatcherSyncer?